notify/__init__.py

```diff
@@ -38,6 +44,8 @@ def notify_unavailable(locations):
         notify_twitter_unavailable_locations(locations)
     if "DISCORD_WEBHOOK_URL" in os.environ:
         notify_discord_unavailable_locations(locations)
+    if "TWILIO_AUTH_TOKEN" in os.environ:
```

Comment:
Eventually I'd like to put these notification methods in classes like `Clinic`, but that can be out of scope for this.